microcosm_metrics/decorators.py

```diff
@@ -58,9 +58,19 @@ def wrapper(*args, **kwargs):
                     return func(*args, **kwargs)
                 finally:
                     end_time = time()
+
+                    additional_parameters = dict()
```

Comment:
Rather than define this as an iterable mapper, how about a callable that returns `tags`? We can default to a function that returns none...